packages/deployer/serverless.aws.yml

```diff
@@ -0,0 +1,39 @@
+service:
+  name: airnode-${opt:providerIdShort}
+
+plugins:
+  - serverless-plugin-typescript
+  - serverless-plugin-optimize
+
+provider:
+  name: aws
+  region: ${opt:region}
+  stage: ${opt:stage}
+  runtime: nodejs12.x
+  memorySize: 256
+  environment: ${file(secrets.json)}
+  iamRoleStatements:
+    - Effect: 'Allow'
+      Action: 'lambda:InvokeFunction'
+      Resource: '*'
```

Comment:
All functions are allowed to invoke all functions of the user

Reply:
Yeah it's possible, I just didn't look into it further.

Reply:
Ok I think it's worth doing it before merging or logging an issue for this. It will be pretty easy to forget about later on. What do you think?

Reply:
Agreed https://github.com/api3dao/airnode/issues/142